### DISCUSSION TITLE
Finding TF activity for each cluster in TSNE

### DISCUSSION THREAD

**Discussion by Lumskie** [aertslab/SCENIC#245] [R]
Hi all,

I have clustered the data based on TF activity and got various distinct clusters. I would like to look into the difference between the TF activity of each cluster by obtaining a list of TF active in each. How can I extract that data from SCENIC output?

Thanks in advance

**Comment by nasjr08** [comment]
To add to what is askedhere, I was able to do the dimensionality reduction and clustering in a seurat object. I then converted to a SingleCellExperiment and used the findMarkers() function from the scan package. I believe this uses pairwise t-tests by default and I get an output which, when plotted on UMAP plots, differentiate between clusters well. 

What I am unsure about is if this is the best method to do this... For some reason the LogFC output from findMarkers is so low when doing it this method (albeit statistically significant). The highest logFC from this output is 0.2 and goes as low as 0.01 even though the FDR is well below 0.01. Would appreciate any suggestions as to why I get this despite the fact the cluster markers clearly define the separate clusters.

NOTE: When I manually take the activity values from scenic of TFs and compare the logFC between different clusters, the output is higher from the findMarkers function.

Thanks,
Naseer Basma
